core/tools.js

```diff
@@ -1877,7 +1877,9 @@
 				 */
 				border: function( value ) {
 					var ret = {},
-						input = value.split( /\s+/ );
+						// remove every white space after comma from color eg. rbg(10, 20, 30, .75)
```

Comment:
Comment line should start with an uppercased letter and end with a dot.

I suggest going with:

> // Remove every white space after comma from color eg. 'rbg(10, 20, 30, .75)'.